glhe/input_processor/plant_loop_component_factory.py

```diff
@@ -16,15 +16,15 @@ def make_plant_loop_component(comp: dict, ip: InputProcessor, op: OutputProcesso
     inputs = ip.get_definition_object(comp_type, comp_name)
 
     if comp_type == 'flow-profile':
-        return make_flow_profile(inputs, ip, op)
+        return make_flow_profile(inputs, ip)
```

Comment:
The output processor doesn't need to be passed around to these components, so we're just removing it.